# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/alsa-utils/fix-udev-goto-and-ucm-defaults.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- alsa-utils-1.2.13/alsactl/90-alsa-restore.rules.in.vanilla	2024-12-26 19:42:33.623333323 +0100
+++ alsa-utils-1.2.13/alsactl/90-alsa-restore.rules.in	2024-12-26 19:43:30.393333332 +0100
@@ -1,6 +1,6 @@
 # do not edit this file, it will be overwritten on update
 
-ACTION=="add", SUBSYSTEM=="sound", KERNEL=="controlC*", KERNELS!="card*",@extratest@ GOTO="alsa_restore_go"
+ACTION=="add", SUBSYSTEM=="sound", KERNEL=="controlC*", KERNELS!="card*",@extratest@ GOTO="alsa_restore_std"
 GOTO="alsa_restore_end"
 
 ENV{ALSA_CARD_NUMBER}="$attr{device/number}"
@@ -21,8 +21,8 @@
 IMPORT{program}="/usr/bin/cat /run/udev/alsa-hda-analog-card"
 ENV{ALSA_CARD_HDA_ANALOG}!="", ENV{ALSA_CARD_NUMBER}="$env{ALSA_CARD_HDA_ANALOG}"
 
-LABEL="alsa_restore_go"
-TEST!="@daemonswitch@", RUN+="@sbindir@/alsactl@args@ restore $env{ALSA_CARD_NUMBER}"
-TEST=="@daemonswitch@", RUN+="@sbindir@/alsactl@args@ nrestore $env{ALSA_CARD_NUMBER}"
+LABEL="alsa_restore_std"
+TEST!="@daemonswitch@", RUN+="@sbindir@/alsactl@args@ restore -D $env{ALSA_CARD_NUMBER}"
+TEST=="@daemonswitch@", RUN+="@sbindir@/alsactl@args@ nrestore -D $env{ALSA_CARD_NUMBER}"
 
 LABEL="alsa_restore_end"
